### PR TITLE
doc: Update manpages for master (laanwj)

### PR DESCRIPTION
**Pull request by maflcko**
(cherry picked from commit 268c390d02d99a4a93a0a01221e273d2b9695ff7)

This bumps the version to 14.99 in the manpages to mark them "WIP" for master.

### CODE REVIEW

**Comment by fanquake** [comment]
utACK eb49101

File: doc/man/bitcoin-cli.1 [hunk 1]
```diff
@@ -69,16 +71,16 @@ Timeout during HTTP requests (default: 900)
...
 
 This is experimental software.
 Distributed under the MIT software license, see the accompanying file COPYING
-or <http://www.opensource.org/licenses/mit-license.php>.
+or <https://opensource.org/licenses/MIT>
 
 This product includes software developed by the OpenSSL Project for use in the
-OpenSSL Toolkit <https://www.openssl.org/> and cryptographic software written
-by Eric Young and UPnP software written by Thomas Bernard.
+OpenSSL Toolkit <https://www.openssl.org> and cryptographic software written by
```
Comment (paveljanik) [inline doc/man/bitcoin-cli.1]: Why there are formatting changes likes this one ("by" moved to the previous line)?

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9795/commits/eb4910193accabdb2edf6a361d52050c851de96e

File: doc/man/bitcoin-cli.1 (hunk 1 above)
Comment (laanwj) [inline doc/man/bitcoin-cli.1]: Possibly the formatting algorithm changed from one version of the script to another? It's slightly strange, indeed, but not worrying, diffs like this are to be expected in automatic tooling output.

**Comment by laanwj** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/9795/commits/eb4910193accabdb2edf6a361d52050c851de96e

**Pull request merged**